components/Layout.tsx

```diff
@@ -174,11 +174,13 @@ const MainNavigation = () => {
   )
 }
 
-const MobileNav = () => {
+const MobileNav: React.FC = () => {
```

Comment:
Why adding this modification in this file in this pr for a github workflow? Lets remove this file from it.

Reply:
I think it somehow came up with git push there are no changes in this file required. I have resolved this issue. Please have a look again.